gatsby-config.js

```diff
@@ -1,5 +1,10 @@
 const {JSDOM} = require('jsdom')
 
+const localPlugins = [
+  `gatsby-yummy-categories`,
+  `gatsby-yummy-search-data`,
```

Comment:
Is it some sort of a convention to use backticks in this file instead of quotes?